### DISCUSSION TITLE
Can my Tesla still record while clips are being uploaded?

### DISCUSSION THREAD

**Discussion by stonexcompass** [marcone/teslausb#877] [General]
The car is parked and has access to Wi-Fi and USB shows as unavailable for at least an hour now (trying to upload files I presume?). Does this mean the car can't record (sentry mode events or manually triggered records for example)?

**Comment by marcone** [comment]
The drive will be disconnected from the car briefly *after* archiving, but should remain available *during* archiving. You can check the archiveloop log to see when TeslaUSB disconnects/reconnects the drive from the car.
